api/src/main/java/org/openmrs/module/episodes/service/EpisodeAttributeTypeService.java

```diff
@@ -0,0 +1,31 @@
+package org.openmrs.module.episodes.service;
+
+import org.openmrs.ConceptAttributeType;
+import org.openmrs.annotation.Authorized;
+import org.openmrs.module.episodes.EpisodeAttributeType;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.validation.constraints.NotBlank;
+import java.util.List;
+
+public interface EpisodeAttributeTypeService {
+    @Transactional(readOnly = true)
+    @Authorized({"Get Episode Attribute Types"})
+    List<EpisodeAttributeType> getAllAttributeTypes(boolean includeRetired);
+
+    @Transactional(readOnly = true)
+    @Authorized({"View Appointments"})
```

Comment:
Seems like an incorrect privilege mapped. Should be `Get Episode Attribute Types`